services/gem/gem-downloads.service.js

```diff
@@ -122,22 +122,39 @@ module.exports = class GemDownloads extends BaseJsonService {
     return [
       {
         title: 'Gem',
-        previewUrl: 'dv/rails/stable',
+        exampleUrl: 'dv/rails/stable',
+        urlPattern: 'dv/:package/stable',
+        staticExample: this.render({
```

Comment:
I've switched to generate the labels using out existing code for doing so in 1a5d94a instead of hard-coding it.

Side note: 'stable' is not a 'tag'  in this situation. It is a special-case of version: https://github.com/badges/shields/blob/bee965b7764c1d9f15139707d1c374f2f896dc28/services/gem/gem-downloads.service.js#L78